docs/advanced/settings.md

```diff
@@ -342,45 +383,36 @@ Default: ""
 }
 ```
 
-## `ui-index`
-
-This setting allows you to configure HTML index location for the UI.
+## `support-bundle-image`
 
-Default: `https://releases.rancher.com/harvester-ui/dashboard/latest/index.html`
+_Available as of v1.2.0_
 
-#### Example
+This setting allows you to configure the support bundle image, with various versions available in [rancher/support-bundle-kit](https://hub.docker.com/r/rancher/support-bundle-kit/tags).
 
+Default:
 ```
-https://your.static.dashboard-ui/index.html
+{
+  "repository": "rancher/support-bundle-kit",
+  "tag": "v0.0.25",
+  "imagePullPolicy": "IfNotPresent"
+}
 ```
 
-## `ui-plugin-index`
-
-This setting allows you to configure the JS address for the Harvester plugin (when accessing Harvester from Rancher).
+## `support-bundle-namespaces`
 
-Default: `https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd.min.js`
-
-#### Example
-
-```
-https://your.static.dashboard-ui/*.umd.min.js
-```
+_Available as of v1.2.0_
 
-## `ui-source`
+This setting allows you to specify additional namespaces when collecting a support bundle. If set to "none," the support bundle will only capture resources from default system namespaces.
```

Comment:
`from default system namespaces.` -> `from the default namespace`

system seems redundent